<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CaptureApplication</class>
 <widget class="QWidget" name="CaptureApplication">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>460</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>460</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CaptureApplication</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_normal">
      <attribute name="title">
       <string>常规</string>
      </attribute>
      <widget class="QPushButton" name="pushButton_open">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>打开</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_shortcut">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>截屏</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>321</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>复制时使用md格式</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="autoRuncheckBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>321</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>开机自启动</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_config">
      <attribute name="title">
       <string>配置</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>201</width>
         <height>24</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>语言：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_langue"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
